### PR TITLE
Nausheen : Review the Code

### PR DESCRIPTION
**Pull request by Nausheenbelim**

### CODE REVIEW

**Comment by linuxeasy** [comment]
rejected

**Comment by Nausheenbelim** [comment]
Good Job.. Fantastic

**Comment by Nausheenbelim** [comment]
good

**Comment by Nausheenbelim** [comment]
huh

File: pom.xml [hunk 1]
```diff
@@ -122,17 +122,7 @@
 			<myname>Nitin</myName>
 <<<<<<< HEAD
 			<myName>NausheenAkanshaViratNitin</myName>
-			<myName>NausheenAkanshaViratNitin</myName>
-			<myName>NausheenAkanshaViratNitin</myName>
-			<myName>NausheenAkanshaViratNitin</myName>
```
Comment (Nausheenbelim) [inline pom.xml]: good Job...Akkal aagyi!!

**Comment by Nausheenbelim** [comment]
Akansha Please review

File: pom.xml (hunk 1 above)
Comment (HarshadMasvekar) [reply pom.xml]: kisko

**Pull request merged**